test/api/xrpToDrops/index.ts

```diff
@@ -8,11 +8,11 @@ import {TestSuite} from '../../utils'
  * - Check out "test/api/index.ts" for more information about the test runner.
  */
 export default <TestSuite>{
-  'works with a typical amount': function(api) {
+  'works with a typical amount': function (api) {
```

Comment:
I think/hope that when we merge `develop`, a lot of these formatting diffs will disappear